tests/functional/v2/task/testReindex.js

```diff
@@ -0,0 +1,89 @@
+const assert = require('assert');
+const uuid = require('uuid');
+const { mpuBucketPrefix } = require('arsenal/lib/constants');
```

Comment:
`require('arsenal').constants`

Reply:
My editor likes to "fix" them for me :face_with_head_bandage: